packages/chrome/src/styled/body/StyledContent.tsx

```diff
@@ -6,15 +6,25 @@
  */
 
 import styled from 'styled-components';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
-import ChromeStyles from '@zendeskgarden/css-chrome';
+import stripUnit from 'polished/lib/helpers/stripUnit';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'chrome.content';
 
 export const StyledContent = styled.div.attrs({
```

Comment:
I noticed that this file doesn't contain JSX, should the file extension be a `.ts` file rather than `.tsx` file, even though the `.tsx` file would work?

Reply:
Good catch. Moved to `*.ts`